packages/tpetra/scripts/PerformanceTesting/vortex/loadEnv.sh

```diff
@@ -0,0 +1,6 @@
+module load sparc-dev/cuda-11.2.152_gcc-8.3.1_spmpi-rolling
+export NVCC_WRAPPER_TMPDIR=/tmp
```

Comment:
Recommend `/tmp/${USER}` instead (though also remember to mkdir that directory as well), we've seen problems in the past with the nvidia compiler's temp files having hash collisions between users depending on how heavily the machine is used.  That's actually exactly why I added the NVCC_WRAPPER_TMPDIR capability way back when.

Reply:
@sebrowne That is a good point. I just tried keeping the ``/tmp/${USER}`` value from sparc-dev, and the only reason it failed is because ``/tmp/${USER}`` may not exist. Pretty sure that's the only reason I added this line.

But I can just ``mkdir -p /tmp/${USER}`` here instead. I will update this